extensions/interactions/CodeRepl/protractor.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for the Pencil Code
+ * Editor.
+ */
+
+var waitFor = require('../../../core/tests/protractor_utils/waitFor.js');
+
+var customizeInteraction = function(interactionEditor, placeHolderText) {
+  browser.executeScript(
+    "var editor = $('schema-based-editor .CodeMirror')[0].CodeMirror;" +
+    "editor.setValue('" + placeHolderText + "');");
+};
+
+var expectInteractionDetailsToMatch = function(elem, placeHolderText) {
+  expect(
+    elem.element(by.css('.CodeMirror')).isPresent()
+  ).toBe(true);
+  // The /n must be included in the check because the editor inserts a newline.
+  // For testing purposes it is required that the order of
+  // the quotes is single-quotes within double-quotes
+  /* eslint-disable quotes */
+  var testValue = browser.executeScript(
+    "var elem = $('.protractor-test-preview-tab .CodeMirror')[0].CodeMirror;" +
+    "return elem.getValue()");
+  /* eslint-enable quotes */
+  expect(testValue).toEqual(placeHolderText + '\n');
+};
+
+var submitAnswer = function(conversationInput, answerCode) {
+  browser.executeScript(
+    "var elem = $('.protractor-test-preview-tab .CodeMirror')[0].CodeMirror;" +
```

Comment:
Don't you need to disable quotes here too?

Reply:
@nithusha21 I am not sure why this didn't create any problem with linting.